src/optiondefs.h

```diff
@@ -2978,7 +2978,7 @@ static struct vimoption options[] =
     {"wlseat",	    "wse",  P_STRING|P_VI_DEF,
 #ifdef FEAT_WAYLAND
 			    (char_u *)&p_wse, PV_NONE, did_set_wlseat, NULL,
-			    {(char_u *)"", (char_u *)0L}
+			    {(char_u *)NULL, (char_u *)0L}
```

Comment:
that change looks wrong and unrelated. I think it also causes the test failures.